src/PlutotvData.cpp

```diff
@@ -262,6 +263,16 @@ std::string PlutotvData::GetSettingsUUID(const std::string& setting)
   return uuid;
 }
 
+int PlutotvData::GetSettingsStartChannel()
+{
+  int startnum = 1;
+
+  if(!kodi::addon::CheckSettingInt("start_channelnum", startnum))
+    kodi::Log(ADDON_LOG_DEBUG, "Start channel number not set, defaulting to 1");
```

Comment:
The `if` and the log line can be removed, as not having the number set is something that should never happens as the setting definition sets a default value and forbids "empty" values.